### PR TITLE
Q2 updates

### PR DESCRIPTION
**Pull request by chomipi88**

### CODE REVIEW

File: astro/data-protection.md [hunk 1]
```diff
@@ -24,3 +24,9 @@ Each Cluster in your Data Plane has its own certificates which were generated wh
...
+
+## Deployment Network Isolation
+
+All deployment namespaces on Astro, including those running on the same cluster, are network isolated from each other by default. In addition to the isolation between deployment namespaces, we also restrict communications within a namespace to only required communication between components and associated ports. This level of network isolation is achieved using [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) enabled by the [Calico](https://kubernetes.io/docs/concepts/cluster-administration/networking/#calico) kubernetes network plugin.
```
Comment (paolaperaza) [inline astro/data-protection.md:30]:
```suggestion
All infrastructure specific to a single Deployment on Astro is isolated to a corresponding Kubernetes namespace within the Astro Cluster in which the Deployment is hosted. All Deployment namespaces on Astro, including those running on the same Cluster, are network isolated from each other by default. In addition to the isolation between Deployment namespaces, we also restrict communication within a namespace to only what is required between components and associated ports.

This level of network isolation is achieved using [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) enabled by the [Calico](https://kubernetes.io/docs/concepts/cluster-administration/networking/#calico) network plugin for Kubernetes.
```

Comment (paolaperaza) [inline astro/data-protection.md:30]:
```suggestion
All infrastructure specific to a single Deployment on Astro is isolated to a corresponding Kubernetes namespace within the Astro Cluster in which the Deployment is hosted. All Deployment namespaces on Astro, including those running on the same Cluster, are network isolated from each other by default. In addition to the isolation between Deployment namespaces, we also restrict communication within a namespace to only what is required between components and associated ports.

This level of network isolation is achieved using [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) enabled by the [Calico](https://kubernetes.io/docs/concepts/cluster-administration/networking/#calico) network plugin for Kubernetes.
```

File: astro/data-protection.md [hunk 2]
```diff
@@ -24,3 +24,9 @@ Each Cluster in your Data Plane has its own certificates which were generated wh
...
+
+The network isolation between and within deployment namespaces ensures communication is restricted to only allow necessary communications within a namespace, and communication between deployments is denied, ensuring unintended communications and attempted data exchanges are blocked.
```
Comment (paolaperaza) [inline astro/data-protection.md:32]:
```suggestion
The network isolation between and within Deployment namespaces ensures that communication is restricted to only allow necessary communications within a namespace, and that communication between Deployments is denied. This ensures that unintended communications and attempted data exchanges are blocked.
```

Comment (paolaperaza) [inline astro/data-protection.md:32]:
```suggestion
The network isolation between and within Deployment namespaces ensures that communication is restricted to only allow necessary communications within a namespace, and that communication between Deployments is denied. This ensures that unintended communications and attempted data exchanges are blocked.
```

**Review by paolaperaza — APPROVED** [review]
Minor copy suggestions but LGTM @chomipi88 !

File: astro/data-protection.md (hunk 1 above)
Comment (chomipi88) [reply astro/data-protection.md]: committed with a minor edit.
replaced infrastructure with 'pods and services', as we don't dedicate infrastructure (eg. nodes) to a namespace.

**Pull request merged**